documentation/modules/running-migration-plan.adoc

```diff
@@ -21,19 +21,40 @@ The *Plans* list displays the source and target providers, the number of virtual
 . Click *Start* beside a migration plan to start the migration.
 . Click *Start* in the confirmation window that opens.
 +
-The *Migration details by VM* screen opens, displaying the migration's progress
+The plan's *Status* changes to *Running* and migration's progress is displayed.
 +
 Warm migration only:
 
 * The precopy stage starts.
 * Click *Cutover* to complete the migration.
 
-. If the migration fails:
-.. Click *Get logs* to retrieve the migration logs.
-.. Click *Get logs* in the confirmation window that opens.
-.. Wait until *Get logs*  changes to *Download logs* and then click the button to download the logs.
+. Click the links in the migration's *Status* to see its overall status and the status of each VM:
 
-. Click a migration's *Status*, whether it failed or succeeded or is still ongoing, to view the details of the migration.
+** The link on the left indicates whether the migration failed, succeeded, or is ongoing and reports the number of VMs whose migration succeeded, failed, or was canceled.
+** The link on the right opens the *Virtual Machines* tab of the *Plan Details* page. For each VM, the tab displays the name of the VM, the start and end times of the migration, the amount of data copied, and a progress pipeline for its migration.
+
+. To view your migration's logs, either as it is running or after it is completed:
+
+.. Click the *Virtual Machines* tab.
+.. Click the arrow (*>*) to the left of the virtual machine whose migration progress you want to check.
++
+The VM's details are displayed.
++
+.. In the *Pods* section, in the *Pod links* column, click the *Logs* link.
++
+The *Logs* tab opens.
 +
-The *Migration details by VM* screen opens, displaying the start and end times of the migration, the amount of data copied, and a progress pipeline for each VM being migrated.
-. Expand an individual VM to view its steps and the elapsed time and state of each step.
\ No newline at end of file
+[NOTE]
+====
+Logs are not always available. The most common reasons for this are:
+
+* The migration is from {virt} to {virt}. In this case, `virt-v2v` is not involved, so no pod is required.
+* No pod was created.
+* The pod was deleted.
+* The migration failed prior to running the pod.
+====
+
+.. To see the raw logs, click the *Raw* link.
+.. To download the logs, click the *Download* link.
```

Comment:
Are these optional?

Reply:
Yes, though I think that's obvious from from context. However., since I am not writing a procedure that only describes how to view logs, I'm marking this step's "parent" as optional.